composer.json

```diff
@@ -15,7 +15,8 @@
     "require": {
         "php": "^7.1",
         "psr/http-client": "dev-master",
-        "nyholm/psr7": "^0.3"
+        "nyholm/psr7": "^0.3",
+        "symfony/options-resolver": "^4.0"
```

Comment:
can you do `^3.4 || ^4.0` to be compatible with the Symfony 3.4 LTS ? 4.0 does not add any new features compared to 3.4

Reply:
Yes, of course. Thank you!